test/Grpc.StatusProto.Tests/Grpc.StatusProto.Tests.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.1;net5.0;net472;net6.0;net7.0</TargetFrameworks>
```

Comment:
you might need to update this before merging (we've upgraded to net8 in the meantime).

Reply:
```suggestion
    <TargetFrameworks>net462;net6.0;net7.0;net8.0</TargetFrameworks>
```